generators/client-build/templates/gulp/tasks/watch-client.js.tpl

```diff
@@ -20,10 +21,13 @@ gulp.task('watch-client', () => {
     gulp.watch(config.client.staticJade, ['process-jade']);<% } %>
 
     // watch sass
-    gulp.watch(config.client.styles, ['sass']);<% if (client === 'react') { %>
-    // watch client js
-    lrload.monitor(config.client.dist.path + '/' + config.client.dist.bundle, {displayNotification: true});<% } %>
+    lload.listen();
+    gulp.watch(config.client.styles, ['sass']);
 
+<% if (client === 'react') { %>
```

Comment:
I'm pulling your branch now to see if this lints correctly on a React project. The reason the braces were weird looking was to prevent linting issues.